sample/ios/Podfile

```diff
@@ -1,16 +1,21 @@
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
-platform :ios, '11.0'
+platform :ios, '12.4'
+install! 'cocoapods', :deterministic_uuids => false
 
 target 'sample' do
-
-
   config = use_native_modules!
+
   use_react_native!(
-    :path => config["reactNativePath"],
-    # to enable hermes on iOS, change `false` to `true` and then install pods
-  	:hermes_enabled => true
+    :path => config[:reactNativePath],
+    # Hermes is now enabled by default. Disable by setting this flag to false.
+    # Upcoming versions of React Native may rely on get_default_flags(), but
+    # we make it explicit here to aid in the React Native upgrade process.
+    :hermes_enabled => true,
```

Comment:
H: Enable it for Android as well (Gradle file).

Reply:
Maybe we can do it as a separate PR? Because just flipping the switch doesn't work and this task block other PRs.